ci/build-unix.yml

```diff
@@ -74,19 +74,6 @@ steps:
         platform: '${{parameters.name}}'
         is_release: '${{parameters.is_release}}'
 
-  - bash: |
-      set -euo pipefail
-      cd '$(Build.StagingDirectory)/logs'
-      find . -type f | xargs gzip -9
-      cd -
-      cd bazel-testlogs
-      find . -type f | xargs gzip -9
-
-    displayName: compress logs
-    condition: and(succeededOrFailed(),
-                   or(eq(${{parameters.is_release}}, 'false'),
-                      ne('${{parameters.name}}', 'm1')))
-
```

Comment:
JSON execution logs are created pre-gzipped by Bazel, so there's little benefit to re-gzipping them.